packages/unplugin-vue-i18n/src/core/directive.ts

```diff
@@ -83,11 +83,13 @@ export function directivePlugin({
           if (vuePluginOptions == null) {
             vuePluginOptions = getVuePluginOptions(vuePlugin!)
           }
-          analyzeIdentifiers(
-            getDescriptor(filename, code, vuePluginOptions),
-            vuePluginOptions,
-            translationIdentifiers
-          )
+          if (vuePluginOptions?.compiler) {
+            analyzeIdentifiers(
+              getDescriptor(filename, code, vuePluginOptions),
+              vuePluginOptions,
+              translationIdentifiers
+            )
+          }
```

Comment:
should we reset `vuePluginOptions` if missing compiler?

Reply:
Ah, yeah!
vue-i18n directive does not work without Vue compiler for SSR.

if Vue compiler is not specify on `vitejs/plugin-vue`, `unplugin-vue-i18n` should output warning.
(I recognize `vitejs/plugin-vue` has Vue compiler as defetault)

Reply:
Oh that's true, it is about the v-t directive, I'm going to check if ssr is exposed.

If this is only a vite plugin why not just expose the transform hook inside vite?